### PR TITLE
working branch

### PR DESCRIPTION
**Pull request by obibaadoma**
branch to admission project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            <li><strong>1914 -</strong> Born in Cresco, Iowa</li>
+            <li><strong>1933 -</strong> Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration"</li>
+            <li><strong>1935 -</strong> Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."</li>
+            <li><strong>1937 -</strong> Finishes university and takes a job in the US Forestry Service</li>
+            <li><strong>1938 -</strong> Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants.</li>
+            <li><strong>1941 -</strong> Tries to enroll in the military after the Pearl Harbor attack, but is rejected. Instead, the military asked his lab to work on waterproof glue, DDT to control malaria, disinfectants, and other applied science.</li>
+            <li><strong>1942 -</strong> Receives a Ph.D. in Genetics and Plant Pathology</li>
```
Comment (obibaadoma) [inline index.html:26]: Kindly refuse to use the "strong "key word" but rather place it in a class and get a css for it.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main id="main">
+        <header id="title">Dr. Norman Borlaug</header>
+        <p>The man who saved a billion lives</p>
+        <figure id="img-div">
+        <img id="image"  src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg">
```
Comment (obibaadoma) [inline index.html:15]: Please use .png file as it is lighter than png file. It will save a lot space in your work.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="styles.css">
```
Comment (obibaadoma) [inline index.html:8]: Please link the css to html, as difference in name not rendering the css as not useful.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    font-size: 50px;
+}
+figure {
+    align-items: center;
+    background-color: white;
+}
+img{
+    display: block;
+}
+#image {
```
Comment (obibaadoma) [inline styles.css:20]: One of the .image {
} needs to be deleted as it may conflict with the rendering of project.

**Review by obibaadoma — COMMENTED** [review]
# You are almost there, please fix linters errors in both html and css. 
# Link html page with css and change the image used to png.

Please do well to include a readme document to the code as well.

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED  
 ## Hi @obibaadoma ,
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 - Used GitHub's 'Review changes' feature from GitHub :heavy_check_mark: 
- Utilised GitHub's inline comments feature for reviews :heavy_check_mark: 
 
 Now you can merge this branch and move on :1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers four_leaf_clover:


 
 ## Optional suggestions
- [ ] You can use a couple of emojis and/or gifs to make the review more human-friendly. I can see you tried to add a few of them but they are not working. Kindly check and make sure they are. :+1:  :wink:  :smile: 

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**